### PR TITLE
Feature/#2013 user profile

### PR DESCRIPTION
**Pull request by aminbksi**

### CODE REVIEW

File: src/scenes/widgets/pages/ProfileWidget/components/templates/UserProfileView/UserProfileView.tsx [hunk 1]
```diff
@@ -85,7 +85,7 @@ const UserProfileView: FC<PropsInterface> = ({showUserInteractions, onClose, use
       </styled.Actions>
       <styled.Details>
         {userProfile?.profile?.bio && (
-          <Text text={userProfile.profile.bio} size="xs" align="left" />
+          <Text text={userProfile.profile.bio} size="xs" align="left" breakLongWord />
```
Comment (PAKovalev) [inline src/scenes/widgets/pages/ProfileWidget/components/templates/UserProfileView/UserProfileView.tsx:88]: No need to create a new prop. Use existing `isMultiline={false}`

File: src/ui-kit/organisms/Dialog/Dialog.tsx [hunk 2]
```diff
@@ -66,7 +66,7 @@ const Dialog: FC<DialogPropsInterface> = ({
   iconSize,
   isBodyExtendingToEdges,
   showBackground = true,
-  controlUnityKeyboardControll = false,
+  controlUnityKeyboardControl = false,
```
Comment (PAKovalev) [inline src/ui-kit/organisms/Dialog/Dialog.tsx:69]: maybe just `keyboardControl`

**Review by PAKovalev — CHANGES_REQUESTED** [review]

File: src/scenes/widgets/pages/ProfileWidget/components/templates/UserProfileView/UserProfileView.tsx (hunk 1 above)
Comment (aminbksi) [reply src/scenes/widgets/pages/ProfileWidget/components/templates/UserProfileView/UserProfileView.tsx:88]: it is different from multiline, it tries to fit the long words in two lines or more

Comment (PAKovalev) [reply src/scenes/widgets/pages/ProfileWidget/components/templates/UserProfileView/UserProfileView.tsx:88]: Just try )

**Review by PAKovalev — APPROVED** [review]

**Review by dmitry-yudakov — APPROVED** [review]

**Pull request merged**